src/components/Custom/Forms/ProductFiltersForm/ProductFiltersForm.tsx

```diff
@@ -41,6 +40,10 @@ const ProductFiltersForm = ({handleSearchParamsChange}: ProductFiltersFormProps)
       brands: brands
     }
   });
+  const {priceRange, ratingRange, category: categoryFromInput, brands: brandsFromInput} = getValues();
+  const [selectedCategory, setSelectedCategory] = useState<Category>(categoryFromInput);
+  const [selectedBrands, setSelectedBrands] = useState<Brand[]>(brandsFromInput as unknown as Brand[]);

```

Comment:
Why do you need these states?

Reply:
To re-render. Left is as it is for now

Reply:
Fixed on next PR ✔: [Task-41: BE: Extend GET all products API with search](https://github.com/SoliaMarko/freshnesecom-store-client/pull/32 )